features/facebookImport/test/ministory/about-pictures-data-analysis.test.js

```diff
@@ -60,3 +64,30 @@ describe("Pictures ministory on export with JPEG pictures in correct location",
         expectActiveAnalysis(analysis);
     });
 });
+
+describe("Pictures ministory on export with JPEG pictures in posts/media location", () => {
```

Comment:
I don't think two different tests are needed. Just add a couple of entries or more.

Reply:
I'd still like to have two tests as normally we will not have exports with both paths. So then a test for each type of export.